portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -46,9 +47,14 @@ public void doGet(HttpServletRequest request, HttpServletResponse response) thro
     DatastoreService datastore = DatastoreServiceFactory.getDatastoreService();
     PreparedQuery results = datastore.prepare(query);
     for (Entity entity : results.asIterable()) {
+      /* Convert the comment's timestamp (ms since Epoch) to a date (d/m/y) */
+      SimpleDateFormat sdf = new java.text.SimpleDateFormat("MMMM d, yyyy");
+      sdf.setTimeZone(java.util.TimeZone.getTimeZone("GMT-4")); /*EDT*/
```

Comment:
maybe you should use getDefault (https://www.geeksforgeeks.org/timezone-getdefault-method-in-java-with-examples/) to get the local timezone)